lib/index.js

```diff
@@ -203,6 +203,17 @@ function lowerCaseObjectKeys(obj) {
   return newObject;
 }
 
+/**
+ * Takes in a list of covered content and a request
+ * and turns them into a canonicalized signature string.
+ *
+ * @param {Object} [options = {}] - Options to use.
```

Comment:
```suggestion
 * @param {object} [options = {}] - Options to use.
```
All of these should be lowercase-o `{object}`, from what I understand.

Reply:
the eslint rule is javascript primitives are lower-case so `boolean`, `object`, `number`, `symbol`, `string` and then capitalize data structures composed from the primitives such as `Array` and `Function`. So yeah you are correct, but the linter is not throwing here... so I upgraded the linter and then linted all the files too. Thanks for catching this as the linter is a bit behind: https://github.com/digitalbazaar/http-signature-header/pull/24/commits/14216760f2d9028f177f3bfb3851ef9ee15e75e4